Actions improvements (#3)
raunc/eutpop-ref-sheet: src/actions.tex

\actionHeadingNoSpace{Event \normal{(p.~12)}}
\begin{itemize}
	\item Must be taken exactly once per Round
	\item Must select a face-up Event
\end{itemize}
\begin{enumerate}

> 	\item Collect \ducats on the card and move Round Status marker

Reviewer: ```suggestion
\end{itemize}
\begin{enumerate}
	\item \strong{Select} a face-up Event, \strong{collect} \ducats on the card and \strong{move} Round Status marker
```
In the rulebook "Select a face-up Event" is in step 1. I'd do the same here and highlight verbs in this sentence to make it more visible that there are 3 activities in this step. This seems to save 1 line, which would be enough to put back "Apply immediate effects" in Research Action below

		\subsubsection*{Standard Event effects \normal{(p.~40)}}
		\smallheading{Gain/place Influence}
		\begin{itemize}

> 			\item 1+ Province in Area must be Owned by NPR/Vassal

Reviewer: Might be already implied by other rules, but I'd still keep it here as a reminder. Seems like there is enough space. If it doesn't fit, then I'd consider shortening statements about Research Idea (Saying "If the Idea is not on display, replace unpicked non-Basic Idea of the same type" in a single point instead of two seems to free up a line)

		\end{itemize}
		\smallheading{Gain Royal Marriage}
		\begin{itemize}

> 			\item Target must be eligible

Reviewer: Same as above. Might be already implied by other rules, but I'd still keep it here as a reminder. If not enough space, then shortening Research Idea statement might help

				\item If DNPR has a Port facing any Main Map Sea Zone without */\textdagger, \conj{and} it is Age II or later
				\begin{itemize}
					\item May place the \dnpr on vacant Territor.

> 					\item \hspacezero\zsavepos{arrowdistantTradeStartThree}If first \dnpr on a Distant Continent, add Trade Cards

Reviewer: ```suggestion
					\item \hspacezero\zsavepos{arrowDistantTradeStartThree}If first \dnpr on a Distant Continent, add Trade Cards
```
Typo in the variable name. Not very important, feel free to ignore

	\item Pay 1\diplopower for a private 3-minute conference
	\item All other communication must be public
\end{itemize}

\actionHeading{Research Idea \normal{(X\monarchpower) (p.~12)}}
\begin{itemize}
	\item Mark it with a Tag

> 	\item Score \p2

Reviewer: ```suggestion
	\item Mark it with a Tag
	\item Apply immediate effects
	\item Score \p2
```
Let's re-add this, if possible. It seems possible to make room for it (see the comment above about selecting an Event). If it still does not fit, then feel free to ignore this comment. I might try to fit it later in case I have time, but not very important actually

	\item If ≤2 other PRs tagged it, they score \p1

> 	\item If you \action{Research} another Government Form Idea while having one (p.~21)

Reviewer: ```suggestion
	\item If you \action{Research} another \strong{Government Form} Idea while having one (p.~21)
```
Nice that rules about Government Forms are here now. I think it would be even better if highlighted. Then it is easier to quickly see that this part of the text can be skipped when not Researching Government Forms

	\begin{itemize}
			\item Lose the previous one (no \prestige loss)
			\item Score \prestige as normal for the new one
	\end{itemize}

> 	\item If \authority ≥ 3, Imperial Subjects may not \action{Research} Government Form Ideas (p.~44)

Reviewer: ```suggestion
	\item If \authority ≥ 3, \strong{Imperial Subjects} may not \action{Research} Government Form Ideas (p.~44)
```
I'd highlight "Imperial Subjects" or, alternatively, "Government Form" to make it easier to see that this can be skipped most of the time. (Highlighting both would probably be too much)

		\item Your Realm, \conj{or}
		\item If having \idea{QftNW}, your \ship
	\end{itemize}

> 	\item Place Colonial \claim on Discovered Area

Reviewer: ```suggestion
	\item Place \claim on Discovered Area
```
It would not be "Colonial" in case the Discovered Area contains NPR or PR Province.
However, it seems useful to have a definition of Colonial Claim somewhere. Perhaps the added definitions list in the beginning of Other Rules pages would be a good place for it

	\item Cath. Realms have Holy War CB against Muslim Realms Owning Prov. in the Area	
\end{itemize}

\subsectionstriped{colorHre}{HRE Actions}
\actionHeadingNoSpace{Increase \authority \normal{(1\adminpower + current \authority) (p.~43)}}
\begin{itemize}
	\item You must be the Emperor
	\item Increase \authority by 1
\end{itemize}

\subsectionstriped{colorAdmin}{Admin. Actions}
\actionHeadingNoSpace{Incr. Stab. \normal{(5\adminpower ± current \stability) (p.~14)}}
\begin{itemize}
	\item Increase \stability by 1

> 	\item If Papal Controller, pay -1\adminpower

Reviewer: ```suggestion
	\item If Papal Controller, pay -1\adminpower (p.~46)
```

> 	\item If Orthodox and current \stability is -1/0, pay -1\adminpower (p.~38)

Reviewer: I was also thinking that it might be better to write `1\adminpower less` instead of -1 in places like this (when there is enough room)

Author: I added the change where possible in the actions page.
While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

Reviewer: > I added the change where possible in the actions page. While checking the whole document for this, I spotted some phrasing improvements in the sequence of play. There I left the "receive +/-X" form as it seemed clearer, but I added a couple of details. Let me know if it is ok to squeeze in past reviewed files changes like this if it happens.

Yes, no problem. In case of bigger changes like moving boxes around I'd prefer a separate PR, though

\begin{itemize}
	\item Must Own 1+ Province in Area(s) governed by the Religion slot
	\item All Provinces must be owned by Realms of the same State Religion
	\item If Protestant, pay -1\adminpower (p.~38)
	\item Change Religion to your State Religion
	\item Add 1\unrest to your affected \towns
	\item Roll Rebel Dice in affected Area(s)
\end{itemize}

\subsectionstriped{colorDiplo}{Diplomatic Actions}
\actionHeading{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}
\begin{itemize}
	\item Only with an NPR
	\item Both must be at Peace
	\item Must have at least 2\influence in target Realm
	\item If target is a Distant NPR, then 
	\begin{itemize}

> 		\item must have a \claim on 1+ of target's Areas
> 		\item may use \colonist instead of \influence

Reviewer: ```suggestion
		\item Must have a \claim on 1+ of target's Areas
		\item Use \colonist instead of \influence (p.~32)
```

\begin{itemize}
	\item Must Own 1+ Province in Area(s) governed by the Religion slot
	\item All Provinces must be owned by Realms of the same State Religion
	\item If Protestant, pay -1\adminpower (p.~38)
	\item Change Religion to your State Religion
	\item Add 1\unrest to your affected \towns
	\item Roll Rebel Dice in affected Area(s)
\end{itemize}

\subsectionstriped{colorDiplo}{Diplomatic Actions}

> \actionHeading{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}

Reviewer: ```suggestion
\actionHeadingNoSpace{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}
```
Looks nicer if less space between the higher level heading

\begin{itemize}
	\item Only with an NPR
	\item Both must be at Peace
	\item Must have at least 2\influence in target Realm
	\item If target is a Distant NPR, then 
	\begin{itemize}
		\item Must have a \claim on 1+ of target's Areas
		\item Use \colonist instead of \influence (p.~32)
	\end{itemize}
	\item If target has Alliance, then must have more \influence and remove previous \alliance
	\item Pay \diplopower = ½ of target's Tax Inc. (max 3)
	\item Add \alliance to target's Capital
	\item Add 1\influence to target's Cap. Area
\end{itemize}

> \actionHeadingNoSpace{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}

Reviewer: ```suggestion
\actionHeading{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
```
Does not need "no space" style as it does not come right after higher level heading anymore

\begin{itemize}
	\item Pay 1\diplopower or 3\ducats per \influence (min 1\diplopower)
	\item May place \influence in Areas Adj. to your \towns, \vassals, \alliances, \marriages, \influence that were there at the start of the Turn
	\item Max 2\influence per Area per Turn

> 	\item No \influence on Distant Continents

Reviewer: Seems like there is enough room to put back "Area may contain max 5\influence" clarification

\actionHeading{Trade \normal{(1\diplopower) (p.~15)}}
\begin{itemize}
	\item Must have an available (upright) Merchant
\end{itemize}
\begin{enumerate}
	\item Reveal 3 Trade cards
	\item May collect 2\ducats and end Action
	\item May move 1 Light \ship
	\begin{itemize}

> 		\item Only if, selecting a Trade Node

Reviewer: ```suggestion
		\item Only if selecting a Trade Node where
```
This way it reads like a sentence when combined with sub-points

> \actionHeading{Trade \normal{(1\diplopower) (p.~15)}}

Reviewer: I would move Trade before Influence to shorten the arrow

\begin{itemize}
	\item Must have an available (upright) Merchant
\end{itemize}
\begin{enumerate}
	\item Reveal 3 Trade cards
	\item May collect 2\ducats and end Action
	\item May move 1 Light \ship
	\begin{itemize}
		\item Only if selecting a Trade Node where
		\begin{itemize}
			\item It increases your Trade Power, \conj{or}
			\item It creates a connection to the Node
		\end{itemize}
		\item May displace Opp.'s \ship if no vac. slot
		\item May not enter a Hostile Sea Zone
	\end{itemize}
	\item Select an eligible Trade Node
	\item Select an available (upright) Merchant
	\begin{itemize}
		\item Move it to the selected Node
		\begin{itemize}
			\item Max 1 Merchant per PR (p.~34)
		\end{itemize}
		\item Activate it (lay it on its side)
	\end{itemize}

> 	\item Eligible PRs collect the Trade Income\zsavepos{arrowTradeStart}

Reviewer: ```suggestion
	\item Eligible PRs collect Trade Income\zsavepos{arrowTradeStart}
```

\end{enumerate}

\begin{dynamiccontents*}{frameTradeIncome}\begin{eubox}{\fhTrade}

> 	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}

Reviewer: Made minor wording adjustments and managed to squeeze some more details into this box. Here's the whole contents of the box after changes:

```
	\subsubsection*{\zsavepos{arrowTradeEnd}Trade Income \normal{(p.~34-35)}}
	\begin{itemize}
		\item Human PRs collect Trade Inc. if the Node
		\begin{itemize}
			\item Has their Merchant, \conj{and}
			\item Is \strong{eligible}: connected to Capital Area by \towns, \vassals, \influence or Light \ships (not required in Inland Trade Node's Area; Occupied \towns/\vassals count)
		\end{itemize}
		\item Expanded - red column, otherwise - green
		\item \strong{1 PR per row} may collect Trade Income. Priority (losers are pushed to lower row):
		\begin{enumerate}
			\item Highest Trade Power
			\item Active PR breaks ties as they please
		\end{enumerate}
		\item Each Pirate Ship lowers all PRs by 1 row
	\end{itemize}
	\smallheading{Secondary Trade Nodes}
	\begin{itemize}
		\item If Active PR has highest (may be tied) Trade Power in eligible Sec. Trade Node
		\begin{itemize}
			\item They may collect Income from the lowest row of the Secondary Node
			\item Other players collect income from Primary Node as normal
		\end{itemize}
		\item May not select a Sec. Node with a Pirate
	\end{itemize}
	\smallheading{Trade Power (\tradepower)}
	\begin{itemize}
		\item 1 \tradepower for your Merchant
		\item 1 \tradepower per Key Province you Own
		\item 1 \tradepower per \ship in Adj. Trade Protection slots (only for Maritime Trade Nodes)
		\item Key Prov. with "+" provide +1\tradepower per "+"
		\item Key Prov. do not count for Sec. Node
	\end{itemize}
	{\botrules
	\subsubsection*{Bot Trade \normal{(p.~4)}}
	\begin{itemize}
		\item Bots have 1 Merchant (never exhausted)
		\item When human PR \action{Trades}, Bots collect Trade Income if Node is eligible for them
		\begin{itemize}
			\item Maritime Trade Node (-1\botpower per Pirate)
			\begin{itemize}
				\item Main Map Nodes: 2\botpower
				\item Distant Nodes: 3\botpower
			\end{itemize}
			\item Inland Trade Node: 2\botpower
			\begin{itemize}
				\item Bot must have 1+ Adjacent \town
			\end{itemize}
			\item -1\botpower if hum. PR has 3+ \tradepower from Key Prov. and \idea{Mercantilism} Idea
		\end{itemize}
		\item When Bot collects Trade Income, move its Merchant to that Node	
	\end{itemize}
	}
```

Author: Very nice!

			\end{itemize}
		\end{itemize}
		\item If hum. PR has 3+ \tradepower from Key Prov. and \idea{Mercantilism} Idea, Bots gain 1\botpower fewer
		\item When Bot collects Trade Income, move its Merchant to that Node	
	\end{itemize}
	}
\end{eubox}\end{dynamiccontents*}

\begin{dynamiccontents*}{frameActionCards}\begin{eubox}{\fhActionCards}
	\begin{multicols}{2}
		\subsubsection*{Action Cards \normal{X\monarchpower + Y\ducats (p.~18-19)}}
		\begin{itemize}
			\item Action cost can never go below 0
			\item Cards in hand are secret
			\item Discard piles are public information
			\item If deck is empty when need to draw 
			\begin{itemize}
				\item If >5 cards in discard pile, keep the top 5, shuffle others
				\item If ≤5 cards in discard pile, shuffle all
			\end{itemize}

> 			\item Papal Controller pays -1\adminpower for actions increasing \stability

Reviewer: ```suggestion
			\item Papal Controller pays -1\adminpower for actions increasing \stability (p.~46)
```

> 			\item Papal Controller pays -1\adminpower for actions increasing \stability

Reviewer: The rule about Orthodox ability would also be good to add here. Perhaps there will be enough space when combining the points about countered Covert Action costs into one: "When contered, cost must be payed and Display Card uses spent"

		\end{itemize}
		\smallheading{Covert Actions (\covertaction)}
		\begin{itemize}

> 			\item Can be countered by \action{Counterespion.}

Reviewer: ```suggestion
			\item Can be countered by \action{Counterespionage}
```
Looks like there is enough space on this line now, so it doesn't have to be abbreviated anymore

			\item When countered
			\begin{itemize}
				\item Cost must be payed
				\item Display Card uses must be spent
			\end{itemize}
		\end{itemize}
		\smallheading{Display cards (\displaycard)}
		\begin{itemize}
			\item Max 2 in play per player (must be diff.)
			\item If already having 2 and playing another
			\begin{itemize}
				\item Must discard one in play
			\end{itemize}
			\item Fully charge the card with \cubes when played
			\item 1+ charges are exhausted per use

> 		\end{itemize}

Reviewer: ```suggestion
		\begin{itemize}
			\item Max 2 in play per player (must be diff.)
			\begin{itemize}
				\item Must discard 1 of them to play another
			\end{itemize}
			\item Fully charge the card with \cubes when played
			\item 1+ charges are exhausted per use (in addition to \monarchpower cost)
		\end{itemize}
```
I'd like to clarify that action cost must still be paid, even though cubes are removed from cards. To make room for it, the point about discarding cards can be shortened

			\item If deck is empty when need to draw 
			\begin{itemize}
				\item If >5 cards in discard pile, keep the top 5, shuffle others
				\item If ≤5 cards in discard pile, shuffle all
			\end{itemize}
			\item Papal Controller pays -1\adminpower for actions increasing \stability (p.~46)
			\item If Orthodox and current \stability is -1/0, pay -1\adminpower for actions increasing \stability (p.~38)
		\end{itemize}
		\smallheading{Covert Actions (\covertaction)}
		\begin{itemize}
			\item Can be countered by \action{Counterespionage}
			\item When countered, pay cost and \displaycard uses 
		\end{itemize}
		\smallheading{Display cards (\displaycard)}
		\begin{itemize}
			\item Max 2 in play per player (must be diff.)
			\begin{itemize}
				\item Must discard 1 of them to play another
			\end{itemize}
			\item Fully charge the card with \cubes when played
			\item 1+ charges are exhausted per use (in addition to \monarchpower cost)
		\end{itemize}
		\smallheading{Reactions (\reaction)}
		\begin{itemize}
			\item May be played outside your Action Turn, under specific circumstances
		\end{itemize}
		\smallheading{Card specific rules}
		\begin{itemize}

> 			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \xspace\cavalry (p.~24)

Reviewer: ```suggestion
			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \cavalry (p.~24)
```
Two spaces were rendered with both regular space and `\xspace` before cavalry icon
